mods/cnc/maps/gdi04b/gdi04b.lua

```diff
@@ -144,24 +144,24 @@ WorldLoaded = function()
 	end)
 
 	Trigger.OnEnteredFootprint(AutoTrigger, function(a, id)
-		if not autoTrigger and a.Owner == GDI then
-			autoTrigger = true
+		if not AutoTriggered and a.Owner == GDI then
+			AutoTriggered = true
 			Trigger.RemoveFootprintTrigger(id)
 
 			Build(AutoUnits, true, function(actor)
 				Trigger.OnKilled(actor, NodUnitKilled)
 				IdleHunt(actor)
 			end)
 
-			Trigger.AfterDelay(DateTime.Seconds(4), function()
+			Trigger.AfterDelay(DateTime.Seconds(4), function(tank)
```

Comment:
`Trigger.AfterDelay` doesn't provide any callback args. Should `tank` be defined elsewhere?